iso15118/shared/security.py

```diff
@@ -1305,7 +1306,7 @@ def get_ocsp_url_for_certificate(certificate: Certificate) -> str:
             ExtensionOID.AUTHORITY_INFORMATION_ACCESS
         ).value
     except ExtensionNotFound:
-        logger.exception("Authority Information Access extension not found.")
+        logger.debug("Authority Information Access extension not found.")
```

Comment:
I don't understand your comment in the commit message that having this is optional and not an exception -- since we are doing OCSP validation, we need to know the address of the OCSP responder.

However, this may be worth doing anyway since using `logger.exception` here will just cause the traceback to be logged again -- that would be redundant with logging this if the error is uncaught, or is caught later on and logged.  Is `debug` the right level, though?

Reply:
we discussed this during a face time call